### ISSUE TITLE
Authentication works, with no confirmation dialogs and token/authCode always empty

### ISSUE THREAD

**Issue by creatio-ua** [playgameservices/play-games-plugin-for-unity#1806]
I'm trying to make a Google account authorization for my client-server game. I need to get an access token on the client side to validate it on my server.

Plugin Authenticate call succeeds (the callback gets called) without showing any system confirmation dialogs (never seen any of them). After that, I'm successfully loading friends list, but I have the following calls always returning an empty string:
```
PlayGamesPlatform.Instance.GetIdToken()
PlayGamesPlatform.Instance.GetServerAuthCode() //tried both true/false with same results

```

**What I have**
- play games for unity Plugin version: 0.9.39a
- Unity 5.6.1f1 (win64)
- Firebase Invitations plugin

**Platform config**
```
PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder()
	.RequestEmail()
	.RequestServerAuthCode(false) //this should be unneeded in my scenario, but I tried with it and without it with same result
        .RequestIdToken()
.Build();
```
**what I did in Google consoles**
- linked applications both Android and Web (looks like the web app was added automagically)
- used Client ID from the linked Web Application (not from the Android one)
- used same SHA1 fingerprint everywhere
- testers in both Dev Console and Firebase
- published APK to the Alpha version of the game in dev.console
- published settings on the GPGS console
- in Firebase - Authentication settings turned on Google authentication
I may have forgotten something of what I did, but the authorization dialog just don't want to appear.

**What seems odd**
- The GPGSIds.cs is not created for me, I suspect my setup just don't need it, but I'm not sure.
- No errors or other messages of any kind in logcat from the plugin.
- The user name after authentication is Lerpz (if that is a normal behavior, it is a bit misleading).

Any suggestions would be greatly appreciated, as I have no more ideas how to make it work as intended.

**Comment by amjaliks** [comment]
Don't you forget to call `PlayGamesPlatform.InitializeInstance (config);` before activating the platform?

**Comment by creatio-ua** [comment]
Thanks for trying to help! The whole trouble happened because I used the wrong Client ID! I had a Firebase Invites Client ID "named Web client (auto created by Google Service)"  in the Google API Console, which I've decided to use (it was wrong). 

No adequate error messages were logged. Probably it happened because unity's log handler is tied to our logger, which silently prevents plug-in's logger to log anything.
